tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -39,42 +39,69 @@ jobs:
 
 - template: templates/mac.yml
 
-- template: templates/windows-build.yml
+- template: templates/windows-hosted-build.yml
   parameters:
     Architecture: x64
 
-- template: templates/windows-build.yml
+- template: templates/windows-hosted-build.yml
   parameters:
     Architecture: x86
 
-- template: templates/windows-build.yml
+- template: templates/windows-hosted-build.yml
   parameters:
     Architecture: arm
 
-- template: templates/windows-build.yml
+- template: templates/windows-hosted-build.yml
   parameters:
     Architecture: arm64
 
-- template: templates/windows-build.yml
+- template: templates/windows-hosted-build.yml
   parameters:
     Architecture: fxdependent
 
-- template: templates/windows-build.yml
+- template: templates/windows-hosted-build.yml
   parameters:
     Architecture: fxdependentWinDesktop
 
+- template: templates/windows-packaging.yml
+  parameters:
+    Architecture: x64
+    parentJob: build_windows_x64
+
+- template: templates/windows-packaging.yml
+  parameters:
+    Architecture: x86
+    parentJob: build_windows_x86
+
+- template: templates/windows-packaging.yml
+  parameters:
+    Architecture: arm
+    parentJob: build_windows_arm
+
+- template: templates/windows-packaging.yml
+  parameters:
+    Architecture: arm64
+    parentJob: build_windows_arm64
 
-- template: templates/windows-component-governance.yml
```

Comment:
Is this replaced by
```
task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```
from the `windows hosted build.yml`?

Reply:
yes